Resources/Textures/Objects/Misc/brbsign.rsi/meta.json

```
@@ -0,0 +1 @@
+{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "I made it myself, use or modify it how you please lol.", "states": [{"name": "icon"}]}
```

```suggestion
{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "I made it myself, use or modify it how you please lol.", "states": [{"name": "icon"}]}
```

The copyright should really include your username. Expecting people to dive through the git history to find the original commit to figure out who "I" refers to isn't great.

---

true